project/ProtocolProject.scala

```diff
@@ -44,8 +44,8 @@ object PublicProtocolProject extends AutoPlugin {
       // this repo doesn't exist (yet) but we need this for publishing to Sonatype OSS
       scmInfo := Some(
         ScmInfo(
-          url("https://github.com/lightbend/akkaserverless-protocol"),
-          "scm:git@github.com:lightbend/akkaserverless-protocol.git")),
+          url("https://github.com/lightbend/kalix-protocol"),
+          "scm:git@github.com:lightbend/kalix-protocol.git")),
```

Comment:
This is one is a surprise. Why we are pointing to a non-existent repo in GitHub? See comment on line 44. 

I'm keeping the status-quo, which is faking now that it comes from `kalix-protocol` repo. I think this should just be `kalix-jvm-sdk`, but first I want to dig why we have it.

Reply:
I agree this should be `kalix-jvm-sdk` - I suspect this configuration is a remnant from when the sdk was still part of the (private) akkaserverless-framework repo.

Reply:
ok, I will change this. Makes no sense now.